example/prometheus/main.go

```diff
@@ -60,11 +60,11 @@ func main() {
 		result.Observe(value, labels...)
 	}
 	_ = metric.Must(meter).RegisterFloat64Observer("ex.com.one", cb,
-		metric.WithDescription("A measure set to 1.0"),
+		metric.WithDescription("A valuerecorder set to 1.0"),
 	)
 
-	measureTwo := metric.Must(meter).NewFloat64Measure("ex.com.two")
-	measureThree := metric.Must(meter).NewFloat64Counter("ex.com.three")
+	valuerecorderTwo := metric.Must(meter).NewFloat64ValueRecorder("ex.com.two")
+	valuerecorderThree := metric.Must(meter).NewFloat64Counter("ex.com.three")
```

Comment:
nit

```suggestion
	valuerecorder := metric.Must(meter).NewFloat64ValueRecorder("ex.com.two")
	counter := metric.Must(meter).NewFloat64Counter("ex.com.three")
```

which would propagate below...

Reply:
Done.